pipelines/internal/commands/run/run.go

```diff
@@ -394,17 +395,17 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 			return nil, fmt.Errorf("expanding regions: %v", err)
 		}
 		resources.Regions = regions
-    }
-    if *zones != "" {
+	}
+	if *zones != "" {
 		zones, err := expandPrefixes(project, listOf(*zones), listZones)
 		if err != nil {
 			return nil, fmt.Errorf("expanding zones: %v", err)
 		}
 		resources.Zones = zones
-    }
-    if len(resources.Zones) + len(resources.Regions) == 0 {
-        resources.Zones = []string{"us-east1-d"}
-    }
+	}
+	if len(resources.Zones)+len(resources.Regions) == 0 {
```

Comment:
These look like unrelated formatting changes.  They should be submitted separately

Reply:
Removed changes